README.md

```diff
@@ -1,124 +0,0 @@
-<div align="center">
```

Comment:
Maybe PR shouldn't remove the README.md  ?

Reply:
I attempted to remove it from the PR, but wasn't intending to remove it from the pingCAP's repo! Let me investigate.

Reply:
I corrected this error